### ISSUE TITLE
Alert, Typo in the new Version 1.2.1

### ISSUE THREAD

**Issue by vin-ni** [unjs/rc9#6]
When building Node Application with Version 1.2.1 I get this error:

```
/home/server/mydomain.com/node_modules/rc9/dist/index.cjs:51
  return options?.flat ? config : flat__default.unflatten(config, { overwrite: true });
                 ^

SyntaxError: Unexpected token '.'
```

**Comment by danielroe** [comment]
Looks like you need to upgrade your node version to 14+.

**Comment by pi0** [comment]
Sorry mad bad. Had to happen in a major version. Reverting flat upgrade shortly.

**Comment by pi0** [comment]
BTW please consider upgrading to a support Node.js version. 12.x is going to be dead soon (https://nodejs.org/en/about/releases/)

**Comment by pi0** [comment]
Fixed in `rc9@1.2.2` (https://github.com/unjs/rc9/commit/a4349ab85606e71b65e6a4df70dc889d26223efd)
